Makefile

```diff
@@ -52,6 +52,6 @@ ci.test: ## Run the tests with results formatted for CI
 	docker-compose run web /bin/sh -c 'mkdir $(RESULTS_PATH) && \
 		apk add nodejs yarn && \
 		bundle exec rails assets:precompile && \
-		bundle exec --verbose rspec --format RspecJunitFormatter --out $(RESULTS_PATH)/rspec-results.xml'
```

Comment:
@duncanjbrown You build pipeline will fail as it relies on rspec-results.xml file - see line 99 `https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/azure-pipelines.yml`